### PR TITLE
Update es5-ext

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2306,9 +2306,9 @@
         "less-plugin-inline-urls": "^1.2.0",
         "levn": "^0.4.1",
         "lolex": "^5.1.2",
-        "lru-cache": "^7.7.3",
+        "lru-cache": "^6.0.0",
         "md5": "^2.3.0",
-        "mimic-response": "^4.0.0",
+        "mimic-response": "^3.1.0",
```
Comment (DonJayamanne) [inline package.json:2311]: 4.0.0 was never used anywhere

File: package.json [hunk 2]
```diff
@@ -2306,9 +2306,9 @@
         "less-plugin-inline-urls": "^1.2.0",
         "levn": "^0.4.1",
         "lolex": "^5.1.2",
-        "lru-cache": "^7.7.3",
+        "lru-cache": "^6.0.0",
```
Comment (DonJayamanne) [inline package.json:2309]: 7.7.3 was never used anywhere. & this was throwing errors when running `npm list --production --parseable --depth=99999 --loglevel=error` which is run by `vsce` when building the vsix package

File: package.json (hunk 1 above)
Comment (DonJayamanne) [reply package.json:2311]: this was throwing errors when running `npm list --production --parseable --depth=99999 --loglevel=error` which is run by `vsce` when building the vsix package

File: package.json [hunk 3]
```diff
@@ -2332,12 +2332,11 @@
         "remove-files-webpack-plugin": "^1.4.0",
         "rewiremock": "^3.13.0",
         "rimraf": "^3.0.2",
-        "screenshot-capture": "^0.2.1",
         "screenshot-desktop": "^1.12.7",
         "sinon": "^8.0.1",
         "source-map": "^0.7.3",
         "source-map-support": "^0.5.12",
-        "string_decoder": "^1.3.0",
+        "string_decoder": "~1.1.1",
```
Comment (DonJayamanne) [inline package.json:2339]: 1.3.0 was never used anywhere. & this was throwing errors when running `npm list --production --parseable --depth=99999 --loglevel=error` which is run by `vsce` when building the vsix package

File: package.json [hunk 4]
```diff
@@ -2332,12 +2332,11 @@
         "remove-files-webpack-plugin": "^1.4.0",
         "rewiremock": "^3.13.0",
         "rimraf": "^3.0.2",
-        "screenshot-capture": "^0.2.1",
```
Comment (DonJayamanne) [inline package.json:2335]: This is not used and issues raised against this by Component Gov.

File: package.json [hunk 5]
```diff
@@ -2370,6 +2368,7 @@
         "marked": "4.0.10",
         "simple-get": "3.1.1",
         "url-parse": "1.5.8",
-        "fs-path": "0.0.25"
+        "fs-path": "0.0.25",
+        "es5-ext": "0.10.53"
```
Comment (DonJayamanne) [inline package.json:2372]: Issue raised by component gov

File: package.json [hunk 6]
```diff
@@ -2348,7 +2347,6 @@
         "ts-node": "^8.3.0",
         "tsconfig-paths-webpack-plugin": "^3.2.0",
         "tslib": "^2.3.0",
-        "type-check": "^0.4.0",
```
Comment (DonJayamanne) [inline package.json:2351]: Removing this fixes errors thrown by `npm list --production --parseable --depth=99999 --loglevel=error` which is run by `vsce` when building the vsix package

File: build/azure-pipeline.pre-release.yml [hunk 7]
```diff
@@ -23,6 +23,10 @@ extends:
   template: azure-pipelines/extension/pre-release.yml@templates
   parameters:
     buildSteps:
+      - task: NodeTool@0
+        inputs:
+          versionSpec: '16.13'
```
Comment (DonJayamanne) [inline build/azure-pipeline.pre-release.yml:28]: On CI use node 16 for our builds

File: package.json [hunk 8]
```diff
@@ -2067,7 +2067,6 @@
         "compile-widgetTester": "cross-env NODE_OPTIONS=--max_old_space_size=9096 webpack --config ./build/webpack/webpack.datascience-ui-widgetTester.config.js --watch",
         "kill-compile-webviews-watchd": "deemon --kill npm run compile-webviews-watch",
         "checkDependencies": "gulp checkDependencies",
-        "preinstall": "npx npm-force-resolutions",
```
Comment (DonJayamanne) [inline package.json:2070]: Removed this, every time i ran `npm i`, the package-lock.json would be different, 
now that we're using npm 8, we can use the native `overrides` section.
Now everytime I run npm i, the package-lock is consistent, doesn't change (ever)

File: package.json [hunk 9]
```diff
@@ -2136,7 +2135,7 @@
         "named-js-regexp": "^1.3.3",
         "node-fetch": "^2.6.7",
         "node-stream-zip": "^1.6.0",
-        "pdfkit": "^0.12.1",
+        "pdfkit": "^0.13.0",
```
Comment (DonJayamanne) [inline package.json:2138]: Had to udpate as it was pulling in some stuff that cased ` npm list --production --parseable --depth=99999 --loglevel=error` to fail.

File: package.json [hunk 10]
```diff
@@ -2366,10 +2360,11 @@
         "canvas": "^2.7.0",
         "fsevents": "^2.3.2"
     },
-    "resolutions": {
+    "overrides": {
```
Comment (DonJayamanne) [inline package.json:2363]: Using native npm 8

**Review by amunger — APPROVED** [review]

**Review by rchiodo — APPROVED** [review]

**Comment by rchiodo** [comment]
@DonJayamanne can we merge? I need to have a consistent package-lock.json in the work I'm doing too :)

**Pull request merged**